src/plugins/home/public/application/components/tutorial/instruction_set.js

```diff
@@ -28,7 +28,7 @@ import {
 import * as StatusCheckStates from './status_check_states';
 
 import { injectI18n, FormattedMessage } from '@kbn/i18n-react';
-import { euiThemeVars } from '@kbn/ui-theme';
+import { euiThemeVars } from '@kbn/ui-theme'; // FIXME: remove this, and access style variables from EUI context
```

Comment:
We might be able to be rid of this, if we adopt the css style function callback on L281

Reply:
@eokoneyo that works fine as a solution to remove this import, but unfortunately that has heavy impact to the Jest unit test for the `src/plugins/home/public/application/components/tutorial/tutorial.test.js` file. The test would need to mount the component wrapped in `<EuiProvider>`, which changes the structure of the component, and the tests themselves are using Enzyme and are heavily dependent on the structure of the component.

This entire plugin is on the roadmap to be rewritten into TypeScript in the next quarter. Maybe just adding this FIXME comment is the right thing to do for now?

Reply:
@tsullivan right... it's probably best to resolve this much later then